parl/remote/job.py

```diff
@@ -36,7 +36,7 @@
 from parl.remote import remote_constants
```

Comment:
The actor now prints the log into a temporary file and display it at the front-end.
How about printing the error log at the same way?

Reply:
It will raise some errors when users using the `logging` module. So we will redirect the stderr to the os.devnull.